Docs/src/manual.md

```diff
@@ -885,3 +886,77 @@ One example of this is the Ogre::Light class. It extends AnimableObject and prov
 ## AnimableValue
 
 When implementing custom animable properties, you have to also implement a number of methods on the AnimableValue interface - basically anything which has been marked as unimplemented. These are not pure virtual methods simply because you only have to implement the methods required for the type of value you’re animating. Again, see the examples in Light to see how this is done.
+
+
+@page Instancing Instancing
+
+Instancing significantly reduces the CPU overhead of submitting many separate draw calls and is a great technique for rendering trees, rocks, grass, RTS units and other groups of similar (but necessarily identical) objects.
+
+OGRE supports a variety of techniques to speed up the rendering of many objects in the Scene.
+
+<dl compact="compact">
```

Comment:
this would be a more Doxygeny style:
```
@par @ref Static-Geometry
Pre-transforms and batches up meshes for efficient use as static geometry in a scene.

@par ..
``` 
with the same visual results

Reply:
I'm using the style of the other sections that use the same thing, I don't even know what it is 🤣